docs/Data-Loading-Maintaining-Studies.md

```diff
@@ -58,3 +58,31 @@ You can also remove multiple studies at once by passing the Cancer Study Ids sep
 ./cbioportalImporter.py -c remove-study -id study1_id,study2_id,study3_id
 ```
 Where `study1_id`, `study2_id` and `study3_id` are the Cancer Study IDs of the studies you would like to remove.
+
+## Deleting patient information
+
+To remove a patient from a study, run:
+```
+./cbioportalImporter.py remove-patients --study_ids study1_stable_id --patient_ids patient1_stable_id,patient2_stable_id
+```
+
+You can specify multiple studies separated by comma. Here is an example of removing patient information from two studies.
+```
+./cbioportalImporter.py remove-patients --study_ids study1_stable_id,study2_stable_id --patient_ids patient1_stable_id
```

Comment:
is --patient_ids restricted to one patient in this case? Or can I also have:
` --study_ids study1_stable_id,study2_stable_id --patient_ids patient1_stable_id,patient2_stable_id`? If yes, perhaps consider just adding this as the example, so it is more complete. And the sentence below would then read:
```
The patients with given stable ids have to exist in each study.
```

Reply:
I like the sentence from the docker example commands .md below:
- You can specify multiple studies and/or samples. All sample ids have to exist for each specified study.

Reply:
@pieterlukasse please have a look https://github.com/cBioPortal/cbioportal/pull/10820/commits/9fc5e028abce8d30f3c537d1e18f92a9ad584c71